plans/tester.pp

```diff
@@ -3,10 +3,16 @@
   # get pe_server ?
   $server = get_targets('*').filter |$n| { $n.vars['role'] == 'pe' }
   $agent_linux = get_targets('*').filter |$n| { $n.vars['role'] =~ /agent_linux/ }
-
+  $agent_windows = get_targets('*').filter |$n| { $n.vars['role'] =~ /agent_windows/ }
```

Comment:
can you make the regex  /agent/ then we can just run against all agents, rather than differentiating

Reply:
done